src/events/types/weather/LightningStrikeEvent.ts

```diff
@@ -0,0 +1,53 @@
+import { Cancellable } from "../../Cancellable";
+import { LightningStrikeEventCause } from "./LightningStrikeEventCause";
+import { WeatherEvent } from "./WeatherEvent";
+
+/**
+ * Stores data for lightning striking
+ */
+export class LightningStrikeEvent extends WeatherEvent implements Cancellable {
+    
+    public static getBukkitClasspath(): string {
+        return 'org.bukkit.event.weather.LightningStrikeEvent';
+    }
+   
+    /**
+    * Gets the cause of this lightning strike
+    * @returns strike cause
+    */
+    public getCause():LightningStrikeEventCause {
```

Comment:
Missing a space before the return type.

@connerdouglass I was thinking, we should add eslint and prettier